.gitignore

```diff
@@ -68,3 +68,6 @@ compiled/
 bin/
 include/
 src/
+
+# OSX
+.DS_Store
```

Comment:
Recommend putting these into a global .gitignore and not adding to each repo: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore.